maxitest.gemspec

```diff
@@ -10,7 +10,7 @@ Gem::Specification.new name, Maxitest::VERSION do |s|
   s.license = "MIT"
   s.executables = ["mtest"]
 
-  s.add_runtime_dependency "minitest", [">= 5.0.0", "< 5.17.0"]
+  s.add_runtime_dependency "minitest", [">= 5.0.0", "< 5.18.0"]
```

Comment:
run `rake bundle` after changing this to fix `bundle install` CI step
... also alright to unset the deployment flag in CI

Reply:
right, thanks.

I normally don't upload lock files to allow the  build to fetch the latest version of the gems available.

Reply:
got burnt by that a few times since the project is left without a known good configuration :)
... deployment mode sure is annoying though

Reply:
This is only to prevent spam.  Some people used the CI minutes to mine crypto.
Once someone contribute, the following build are run automatically.